spec/workers/notification_worker_spec.rb

```diff
@@ -232,7 +232,86 @@ def make_it_perform
           tags: { 'topics' => ['foo/bar'] },
           document_type: nil,
           emailing_app: 'email_alert_api',
-          gov_delivery_ids: []
+          gov_delivery_ids: [],
```

Comment:
I think this should assert against the enabled/disabled fields as well for completeness.